module/VuFind/src/VuFind/Config/IniReaderTrait.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/**
+ * Trait for creating INI readers
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Hebis Verbundzentrale 2023.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Config
+ * @author   Thomas Wagener <wagener@hebis.uni-frankfurt.de>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Site
+ */
+
+namespace VuFind\Config;
```

Comment:
We tend to put traits in a Feature sub-namespace -- e..g. VuFind\Config\Feature in this case. I see that we recently added an ExplodeSettingsTrait directly in the VuFind\Config namespace, though. If you prefer to leave this here, that's fine... but if you think separating the traits for cleaner directory organization would be worthwhile, feel free to establish the Feature subdirectory, and I'll open a separate PR to move the other trait in with it.

Reply:
I moved the trait into a new Feature subdir. I can also move the ExplodeSettingsTrait there unless you think it's better to do this in an other PR.

Reply:
Thank you! I think it would be better to move ExplodeSettingsTrait in a separate PR just to avoid creating any confusion here. (Especially since that's a change that can be merged right away without need for much discussion, whereas this is likely going to remain open for a little while longer). If you have time to do it, please feel free. If you don't, let me know and I'll do it when time permits.

Reply:
I created a PR for this.